lib/tasks/github.rake

```diff
@@ -0,0 +1,56 @@
+require 'literate_randomizer'
+require 'octokit'
+
+namespace :github do
+  desc "Sync database authors with those specificed by the Github issues as per Feature-001"
+  task sync: :environment do
+    # Get issues from github, which represent individual 
+    # authors and their associated biography
+    client = Octokit::Client.new
+    client.auto_paginate = true
+    issues = client.issues 'scasagrande/bookstore-authors'
+    
+    # Hash github issue ID and issue data
+    authors_gh_h = Hash.new
+    issues.each do |issue|
+      authors_gh_h[issue[:number]] = issue
+    end
+
+    # Retrieve from the DB the list of authors that have matching names
+    # to our github list of authors.
+    puts "Updating existing authors that have been modified"
+    authors_db_to_keep = Author.where(:github_issue_id => authors_gh_h.keys)
```

Comment:
When working with hashes that have symbol keys, there's an alternative format where you list the key with the collon _after_ it, and then immediately list the value (example below). In latest versions of ruby this is the preferred syntax for symbol-keyed hashes as it's a little easier to read.

```suggestion
    authors_db_to_keep = Author.where(github_issue_id: authors_gh_h.keys)
```